server/authz/policy.rego

```diff
@@ -75,14 +75,14 @@ allow {
 allow {
   object.type == "team"
   object.id != 0
-  team_role(subject, object.id) == [admin,maintainer][_]
+  team_role(subject, object.id) == [admin,maintainer,observer][_]
   action == read
 }
 # or global admins or global maintainers
```

Comment:
Once product confirms, let's change the comment to be `global users can read all teams`

Reply:
Oh yes, thanks for catching that!